app-authz-jakarta-servlet/pom.xml

```diff
@@ -35,55 +35,69 @@
         </license>
     </licenses>
 
+    <properties>
+        <version.keycloak>${project.version}</version.keycloak>
+    </properties>
+
     <dependencies>
         <dependency>
-            <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-core</artifactId>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>6.0.0</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-adapter-core</artifactId>
-            <scope>provided</scope>
+            <artifactId>keycloak-policy-enforcer</artifactId>
+            <version>${version.keycloak}</version>
```

Comment:
Minor: Do we need this `<version>` element? Could it just take it automatically from parent pom (imported from keycloak bom)?